parsl/executors/high_throughput/executor.py

```diff
@@ -25,8 +24,7 @@
     RandomManagerSelector,
 )
 from parsl.executors.high_throughput.mpi_prefix_composer import (
-    VALID_LAUNCHERS,
-    validate_resource_spec,
+    InvalidResourceSpecification,
```

Comment:
this is weird code placement. I made issue #3589

Reply:
I think this is better addressed in a separate PR. What do you think?

Reply:
right, that's what I made the issue for